api/v3/commonservice_types.go

```diff
@@ -54,9 +54,17 @@ type Bedrockshim struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// BedrockOperator maintains a list of bedrock operators
+type BedrockOperator struct {
+	Name    string `json:"name"`
+	Version string `json:"version"`
```

Comment:
@ZhuoxiLi It's better to set Version and Status to `omitempty` for future extension.